config/locales/en.yml

```
@@ -307,28 +307,42 @@ en:
       covid: 'Coronavirus (COVID-19): guidance'
       departments_and_policy_html: Departments and&nbsp;policy
       driving_test: Driving theory test
+      featured: Featured
       find_job: Find a job
-      intro: The best place to find government services and information
+      intro_html: The best place to find government services and&nbsp;information
       intro_simpler: Simpler, clearer, faster
+      intro_title:
+        text: Welcome to GOV.UK
+        html: Welcome to&nbsp;GOV.UK
       job_search: Find a job
       jobseekers_allowance: Jobseeker's Allowance
       merged_websites_html: The websites of all <a href="/government/organisations" class="govuk-link" data-track-category="homepageClicked" data-track-action="departmentsLink" data-track-label="/government/organisations" data-track-value="1" data-track-dimension="government departments" data-track-dimension-index="29">government departments</a> and many other agencies and public bodies have been merged into GOV.UK.
       ministerial_departments: Ministerial departments
       ministerial_departments_count: 23
+      meta_description: GOV.UK - The place to find government services and information - simpler, clearer, faster.
       more: More on GOV.UK
       most_active: Most active
       news: news and communications
       other_agencies: Other agencies and public&nbsp;bodies
       other_agencies_count: 400+
       passport_fees: Passport fees
       popular: Popular on GOV.UK
-      promo_boxes:
-        text_1: Read the latest on the 26th UN Climate Change Conference of the Parties (COP26), hosted in Glasgow.
-        text_2: Get information about vaccinations on NHS.UK.
-        text_3: Check what you need to do to travel internationally.
-        title_1: COP26
-        title_2: COVID-19 vaccinations
-        title_3: 'Travel abroad: step by step'
+      promotion_slots:
+        - text: Read the latest on the 26th UN Climate Change Conference of the Parties (COP26), hosted in Glasgow.
+          title: COP26
+          href: /government/topical-events/cop26
+          image_src: homepage/cop26.jpg
+          srcset: { "homepage/cop26.jpg": "610w", "homepage/cop26-480.jpg": "480w", "homepage/cop26-320.jpg": "320w", "homepage/cop26-240.jpg": "240w", 
+            "homepage/cop26-170.jpg": "170w", }
```

Although using JSON in YAML is okay, could this YAML-ified?

A direct translation would look like this:

```suggestion
          srcset:
            homepage/cop26.jpg: 610w
            homepage/cop26-480.jpg: 480w
            homepage/cop26-320.jpg: 320w
            homepage/cop26-240.jpg: 240w
            homepage/cop26-170.jpg: 170w
```

but this could be the opportunity to make it a bit more readable.

---

Yep agreed that looks much better, i've updated as above.